src/source.js

```diff
@@ -0,0 +1,55 @@
+//Create function to do getData(category) and and getAllLinks() based on paremeter
+
+export function getData(category) {
+
+  if (category) {
```

Comment:
Oh no please do this with ternary operator, basically you copy 2 methods in single, one still repeatable code.

const categoryQuery = category ? `?category=${category}` : '';
Is very simple thing: `https://api.publicapis.org/entries${categoryQuery}`